src/sparseml/pytorch/torchvision/train.py

```diff
@@ -91,13 +92,24 @@ def train_one_epoch(
         start_time = time.time()
         image, target = image.to(device), target.to(device)
         with torch.cuda.amp.autocast(enabled=scaler is not None):
-            output = model(image)
+            outputs = output = model(image)
             if isinstance(output, tuple):
                 # NOTE: sparseml models return two things (logits & probs)
                 output = output[0]
             loss = criterion(output, target)
 
         if steps_accumulated % accum_steps == 0:
+            if manager is not None:
+                loss = manager.loss_update(
```

Comment:
Question: It looks like this is only called once per gradient accum step, instead of once per batch. Is this correct?

Reply:
Yes it needs to be called once per backward pass

Reply:
👍